TheOneEditor/PanelInspector.cpp

```diff
@@ -32,6 +33,7 @@ bool PanelInspector::Draw()
 
 	if (ImGui::Begin("Inspector", &enabled, settingsFlags))
 	{
+
         ImGuiIO& io = ImGui::GetIO();
         ImVec4 clear_color = ImVec4(0.55f, 0.55f, 0.55f, 1.00f);
         ImGui::SetNextWindowSize(ImVec2(250, 650), ImGuiCond_Once);
```

Comment:
Panel Inspector
- Error in vector that has all the components when adding the Script "ExampleScript" (should review from line 332 to 363)